eth/v1alpha1/beacon_chain.proto

```diff
@@ -617,6 +617,8 @@ message ActiveSetChanges {
 message ValidatorPerformanceRequest {
     // A list of 48 byte validator public keys.
     repeated bytes public_keys = 1;
```

Comment:
```suggestion
    repeated bytes public_keys = 1 [deprecated = true];
```
to encourage more usage of indices over public keys eventually, would be best to mark as deprecated as a warning to users of the API